patches/minecraft/net/minecraft/advancements/Advancement.java.patch

```diff
@@ -1,6 +1,23 @@
 --- a/net/minecraft/advancements/Advancement.java
 +++ b/net/minecraft/advancements/Advancement.java
-@@ -309,6 +309,7 @@
+@@ -230,9 +230,13 @@
+          }
+       }
+ 
+-      public Advancement func_203904_a(Consumer<Advancement> p_203904_1_, String p_203904_2_) {
+-         Advancement advancement = this.func_192056_a(new ResourceLocation(p_203904_2_));
+-         p_203904_1_.accept(advancement);
++      public Advancement func_203904_a(Consumer<Advancement> p_203904_1_, String p_203904_2_){
```

Comment:
YOu changed a space here after between the ){ creating 2 patch lines. But I would also suggest to not modify the constructor and simply add the other. This would actually be less patch lines overall in this case.